src/components/CandidateForm.jsx

```diff
@@ -0,0 +1,38 @@
+import React from "react";
+import styles from "./CandidateForm.module.css";
+import Button from "./Button";
+
+
+const CanditateForm = ({inputsData, sendFormData}) => {
+    const inputs = inputsData;
```

Comment:
here you should make use of useState hook. e.g. `const [inputs, setInputs] = useState(inputsData) `.